qml/main.qml

```diff
@@ -1,14 +1,7 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.3
-import QtQuick.Controls.Material 2.1
```

Comment:
Can you remove this commit ?
This files are necessary for windows deployment, otherwise Materials, charts, graphics effects and models will not work when the binary is released with windeployqt.
If you are busy, I can fix that for you.